### PR TITLE
RDoc-2314 SQL ETL: Provide specific parameter types in script + Improve article

### PR DESCRIPTION
**Pull request by Danielle9897**
**Related issue:**
https://issues.hibernatingrhinos.com/issue/RDoc-2314/SQL-ETL-Provide-specific-parameter-types-in-script

---

* Added section: "Loading to specific column types"
* Added section: "Document extensions"
* Added an example for creating the SQL ETL task using the Client API
* General text improvements

### CODE REVIEW

File: Documentation/5.4/Raven.Documentation.Pages/server/ongoing-tasks/etl/sql.markdown [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+### Loading to specific column types
+
+The SQL type of the target column can be explicitly specified in the SQL ETL script.  
+This is done by defining the `Type` and the `Value` properties for the data being loaded.
+
+   * **Type**:  
+     The type specifies the SQL column type the value is loaded to.  
+     The type should correspond to the data types used in the target relational database.  
+     If no type is specified, the column type will be detected automatically.
```
Comment (arekpalinski) [inline Documentation/5.4/Raven.Documentation.Pages/server/ongoing-tasks/etl/sql.markdown:235]: Maybe it's worth mentioning explicitly that the value of the following enums are supported to be provided as type: `SqlDbType, NpgsqlDbType, MySqlDbType, OracleDbType`

Comment (arekpalinski) [reply Documentation/5.4/Raven.Documentation.Pages/server/ongoing-tasks/etl/sql.markdown:235]: Also for some providers it's allowed to combine the values of those enums so we support that via `|` e.g. `Array | Double` for NpgSql

File: Documentation/5.4/Raven.Documentation.Pages/server/ongoing-tasks/etl/sql.markdown [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+        Type: 'Array | Text',
+        Value: this.OrderLines.map(function(l) {return l.Product;})
+    },
+};
+
+// Load the data into the 'Orders' table
+loadToOrders(orderData);
+{CODE-BLOCK/}
+
+* Refer to the official documentation for detailed information on supported types for each relational database:
```
Comment (arekpalinski) [inline Documentation/5.4/Raven.Documentation.Pages/server/ongoing-tasks/etl/sql.markdown:262]: Oh, I see you're referencing specific types here. That's great then

File: Documentation/5.4/Raven.Documentation.Pages/server/ongoing-tasks/etl/sql.markdown (hunk 1 above)
Comment (Danielle9897) [reply Documentation/5.4/Raven.Documentation.Pages/server/ongoing-tasks/etl/sql.markdown]: done - added

**Review by arekpalinski — APPROVED** [review]

**Pull request merged**